setup.py

```diff
@@ -31,7 +31,7 @@
 
 tensorboard_extra_require = [
     "tensorflow-cpu >= 2.3.0, <=2.5.0rc",
-    "grpcio~=1.34.0",
+    "grpcio~=1.37.1",
```

Comment:
Could this be deleted please? `grpcio` is already required through `google-api-core[grpc]`. With pip's new dependency resolution I am concerned that specifying a `grpcio` version here as well will result in version conflicts at some point. https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies

Reply:
Looks like we cannot because TF 2.5 requires 1.34: https://github.com/tensorflow/tensorflow/blob/r2.5/tensorflow/tools/pip_package/setup.py#L129